README.md

```diff
@@ -589,11 +589,19 @@ docker exec -it ees-idp /opt/jboss/keycloak/bin/standalone.sh -Djboss.socket.bin
 Then simply copy the file from the `/tmp/new-ees-realm.json` file in the `ees-idp` container to `src/keycloak-ees-realm.json` in order for future restarts of the IdP to use this new 
 realm configuration.
 
-Aside from unit tests for each project, we maintain suites of Robot Framework tests that can be found in `tests`.
+### Forcing immediate publishing of scheduled Releases in test environments
+
+During manual or automated testing, it is handy to have a way to schedule Releases for publishing but to trigger that process to occur on demand, rather than having to wait for a lengthly
```

Comment:
```suggestion
During manual or automated testing, it is handy to have a way to schedule releases for publishing but to trigger that process to occur on demand, rather than having to wait for a lengthy
```

Reply:
I've spent 43 years of my life saying "lengthly".  Thanks for putting me right at last :joy: